### PR TITLE
Bump chart version and NuoDB version

### PR DESCRIPTION
**Pull request by adriansuarez**
This change bumps the version of NuoDB and enables testing of SQL connectivity via the Ingress controller.

### CODE REVIEW

**Comment by adriansuarez** [comment]
This PR is a another attempt to bump the NuoDB version. PR #353 had to be abandoned because it became unsynchronized from the branch somehow.

Bumping the product version to 5.1 exposed issues with the Ingress tests, which are addressed by bumping the version of the client package.

**Comment by adriansuarez** [comment]
It looks like the issue is persistent and is affecting this PR also, and many others according to https://www.githubstatus.com.

File: test/minikube/minikube_external_access_test.go [hunk 1]
```diff
@@ -354,12 +376,15 @@ func TestKubernetesIngress(t *testing.T) {
...
 
-	// this requires CDriver to support Server Name Indication (SNI); there is
-	// no straightforward way to make this logic conditional because in internal
-	// CI we use the nuosql from the package installation but in CircleCI, we
-	// use the client package
-	// TODO: Change the version once DB-34466 is fixed
-	testlib.RunOnNuoDBVersionCondition(t, ">=5.1", func(version *semver.Version) {
+	// this requires CDriver to support Server Name Indication (SNI)
+	t.Run("sqlIngressConnectivity", func(t *testing.T) {
+		constraint, err := semver.NewConstraint(">=5.0.4")
```
Comment (sivanov-nuodb) [inline test/minikube/minikube_external_access_test.go:381]: Nit: I see that DB-34502 is fixed in 5.0. Should we make the version constraint ">=5.0"?

File: test/minikube/minikube_external_access_test.go [hunk 2]
```diff
@@ -354,12 +376,15 @@ func TestKubernetesIngress(t *testing.T) {
 		}
 	}
 
-	// this requires CDriver to support Server Name Indication (SNI); there is
```
Comment (sivanov-nuodb) [inline test/minikube/minikube_external_access_test.go:357]: Thanks for enabling this!

**Review by sivanov-nuodb — APPROVED** [review]
Looks good.

**Review by agoelzer — APPROVED** [review]

**Review by kontaras — APPROVED** [review]

**Pull request merged**